client/actions/assignments/notifications.js

```diff
@@ -351,6 +352,29 @@ const onAssignmentDeleted = (_e, data) => (
     }
 );
 
+export const onContentUpdate = (_e, data) => (
+    (dispatch, getState) => {
+        const updatedIds = Object.keys(data.items);
+        const currentItems = Object.values(getState().assignment.archive);
```

Comment:
Whenever accessing anything from the redux store, its ideal to use selectors. This way developers don't have to worry about where the data is in the store (or make changes if the store structure changes), all developers have to worry about is what selectors are available to them.